src/components/TiltCard.tsx

```diff
@@ -0,0 +1,192 @@
+import {
+  useRef,
+  useState,
+  useCallback,
+  forwardRef,
+  type CSSProperties,
+} from "react";
+import { classes } from "../utils/classes";
+
+interface ConditionalTiltCardProps extends TiltCardProps {
+  disabled?: boolean;
+}
+
+const ConditionalTiltCard = forwardRef<
+  HTMLDivElement,
+  ConditionalTiltCardProps
+>(function ConditionalTiltCard(
+  {
+    children,
+    disabled = true,
+    maxTilt = 15,
+    glareMaxOpacity = 0.4,
+    className = "",
+    ...extras
+  },
+  ref,
+) {
+  if (disabled) {
+    return (
+      <div ref={ref} className={className} {...extras}>
+        {children}
+      </div>
+    );
+  }
+
+  return (
+    <TiltCard
+      ref={ref}
+      maxTilt={maxTilt}
+      glareMaxOpacity={glareMaxOpacity}
+      className={className}
+      {...extras}
+    >
+      {children}
+    </TiltCard>
+  );
+});
+
+interface TiltCardProps {
+  children: React.ReactNode;
+  maxTilt?: number;
+  glareMaxOpacity?: number;
+  className?: string;
+}
+
+const TiltCard = forwardRef<HTMLDivElement, TiltCardProps>(function TiltCard(
+  { children, maxTilt = 15, glareMaxOpacity = 0.4, className = "", ...extras },
```

Comment:
`className` should be fine to default to `undefined`, rather than an empty string?